esmf-semantic-aspect-meta-model/src/test/java/org/eclipse/esmf/samm/ConstraintShapeTest.java

```diff
@@ -56,7 +71,7 @@ public void testLanguageStringNotUniqueExpectFailure( final KnownVersion metaMod
    }
 
    @ParameterizedTest
-   @MethodSource( value = "allVersions" )
+   @MethodSource( value = "versionsUpToIncluding2_1_0" )
```

Comment:
Why not including 2.2.0?

Reply:
Did you change this because the test fails with the changes in the shape? If so, why does it fail?